xbmc/cores/FFmpeg.h

```diff
@@ -21,6 +22,27 @@ extern "C" {
 #include <libpostproc/postprocess.h>
 }
 
+#include <tuple>
+
+namespace FFMPEG_HELP_TOOLS
+{
+
+struct FFMpegException : public std::exception
+{
+  std::string s;
+  template<typename... Args>
+  FFMpegException(const std::string& fmt, Args&&... args)
+    : s(StringUtils::Format(fmt, std::forward<Args>(args)...))
+  {
+  }
+  ~FFMpegException() throw() {} // Updated
+  const char* what() const throw() { return s.c_str(); }
```

Comment:
`throw()` is deprecated/removed from C++, instead this should be:
```suggestion
  const char* what() const noexcept override { return s.c_str(); }
```

Reply:
Fixed!